wgpu-core/src/lib.rs

```diff
@@ -64,6 +67,31 @@ impl RefCount {
     fn load(&self) -> usize {
         unsafe { self.0.as_ref() }.load(Ordering::Acquire)
     }
+
+    /// This works like std::mem::drop, except that it returns a boolean which is true iff we
```

Comment:
typo "iff"

Reply:
Ah, I actually was using "iff" as [shorthand](https://en.wikipedia.org/wiki/iff) for for "if and only if" but that is probably too esoteric a shorthand! I fixed this.